server/src/gatling/resources/access-tokens.csv

```diff
@@ -0,0 +1,6 @@
+access_token
+79280324-ed93-4760-82fa-90c20f9e8bee
+1edd69ae-896f-4a2d-9187-6bb9a2df3820
+4c6694b2-be08-4dce-928e-180f61fed680
+c17161b1-473f-4ed3-b2e8-c6fc4d027f33
+a7f91dbe-a3a1-4390-aa90-8195339e1191
```

Comment:
For which API are these? Is it safe to publish them here?

Reply:
These are tokens generated for my localhost dev environment.
They function as an example.

Reply:
Ok, could you add that info to the gatling README?